test/librarymanager.test.ts

```diff
@@ -58,8 +61,48 @@ suite("Arduino: Library Manager.", () => {
 
             done();
         }).catch((error) => {
-          done(error);
+            done(error);
         });
     });
 
+    // Arduino: Library Manager: Install extenal libarary.
+    // tslint:disable-next-line: only-arrow-functions
+    test("should be able to Install libraries", function(done) {
+        this.timeout(3 * 60 * 1000);
+        try {
+            // Library Manager: Install extenal libarary.
+            ArduinoContext.arduinoApp.installLibrary("AzureIoTHub", "1.0.35", true).then((result) => {
+                // check if the installation succeeds or not
+                const arduinoSettings = ArduinoContext.arduinoApp.settings;
+                const libPath = Path.join(arduinoSettings.sketchbookPath, "libraries", "AzureIoTHub");
+
+                if (util.directoryExistsSync(libPath)) {
+                    done();
+                } else {
+                    done(new Error("AzureIoTHub library install failure, can't find library path :" + libPath));
+                }
+            });
+
+        } catch (error) {
+            done(new Error(error));
+        }
+    });
+
+    // Arduino: Library Manager: remove extenal libarary.
+    // tslint:disable-next-line: only-arrow-functions
+    test("should be able to remove libraries", () => {
+        try {
+            // Library Manager: remove extenal libarary.
+            const arduinoSettings = ArduinoContext.arduinoApp.settings;
+            const libPath = Path.join(arduinoSettings.sketchbookPath, "libraries", "AzureIoTHub");
+
+            if (util.directoryExistsSync(libPath)) {
+                ArduinoContext.arduinoApp.uninstallLibrary("AzureIoTHub", libPath);
+            }
+

```

Comment:
add assert method to make sure the libpath is removed after calling uninstallLibrary.